java/core/src/main/java/co/worklytics/psoxy/gateway/HttpEventResponse.java

```diff
@@ -14,12 +16,13 @@
  */
 @Value
 @Builder
+@ToString
 public class HttpEventResponse {
 
     int statusCode;
 
     @Singular
-    List<Pair<String, String>> headers;
+    Map<String,String> headers;
```

Comment:
👍  I didn't know that `lombok` `@Singular` was smart enough to work for a `Map<>`